editor/src/utils/common-constants.ts

```diff
@@ -1 +1 @@
-export const styleStringInArray = ['style']
+export const styleStringInArray = ['style'] as ['style']
```

Comment:
this looks a bit wonky but it makes sure that the type is the array literal with a string literal inside it. this means then typescript can pick up this type where we use it to build longer paths!

Reply:
is this the same as suing `as const`?

Reply:
`as const` makes it a `ReadonlyArray<'style'>` and quite a few places complained that they take Array and not ReadonlyArray, and I didn't want to inflate the scope of this PR